src/test/java/com/code_intelligence/jazzer/mutation/mutator/aggregate/CachedConstructorMutatorTest.java

```diff
@@ -192,4 +192,34 @@ void testEmptyArgsConstructor() throws IOException {
     EmptyArgs read = mutator.readExclusive(new ByteArrayInputStream(new byte[] {}));
     mutator.writeExclusive(read, new ByteArrayOutputStream());
   }
+
+  static class GenericClass<T> {
+    private final T t;
+
+    GenericClass(T t) {
+      this.t = t;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
We don't need `equals` and `hashCode` methods if the object instances are never compared to each other.